thunder/tests/opinfos.py

```diff
@@ -2496,6 +2496,31 @@ def addcmul_addcdiv_sample_generator(op, device, dtype, requires_grad, **kwargs)
 opinfos.append(addcdiv_opinfo)
 
 
+def lerp_sample_generator(op, device, dtype, requires_grad, **kwargs):
+    S = 4
+    cases = (
+        ((S,), (S,), (S,)),
+        ((S, S), (S, S), (S, S)),
+        ((S, 1), (1, S), (S, 1)),
+    )
+    make = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
+    number = partial(make_number, dtype=dtype)
+    for s0, s1, s2 in cases:
+        yield SampleInput(make(s0), make(s1), make(s2))
+        weight = number(**kwargs)
```

Comment:
Add a comment describing the two cases (one where weight is a tensor with the shape described above, another where it's a number)

Reply:
May I ask what kind of comments you intend? The names `s0, s1, s2` were unhelpful, but I did not find cognitive burden in these `SampleInput`s.

Reply:
```
# Generates two cases, one with a tensor weight using the shape from the case, one with a number weight
```